lib/valueflow.cpp

```diff
@@ -8077,7 +8071,7 @@ static void valueFlowContainerSize(TokenList* tokenlist,
         }
         for (const ValueFlow::Value& value : values) {
             if (constSize)
-                valueFlowForwardConst(var->nameToken()->next(), var->scope()->bodyEnd, var, values, settings);
+                valueFlowForwardConst(var->nameToken()->next(), var->scope()->bodyEnd, var, values, settings); // TODO: is "values" correct here?
```

Comment:
Actually I think this branch can be moved out of the loop.

Reply:
Will do that.